os/board/rtl8730e/src/rtl8730e_boot.c

```diff
@@ -240,6 +240,7 @@ void board_gpio_initialize(void)
 		u32 pinpull;
 	} pins[] = {
 				{PA_23, PIN_INPUT, PullNone},
+				{PB_20, PIN_OUTPUT, PullNone},
```

Comment:
Let's set it as pd.

Reply:
it is already defined as PB by realtek in bsp code

Reply:
Oh, I mean `pulldown`, not `PD_20`.